spec/observableBehaviors.js

```diff
@@ -10,6 +10,11 @@ describe('Observable', function() {
         expect(ko.isObservable(instance)).toEqual(true);
     });
 
+    it('Should advertise that instances are not observable', function () {
```

Comment:
Thanks for the PR;  Perhaps this would be better labelled `instances` should be `constructors`?